internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -113,6 +113,69 @@ func entityFromWebhookEventTypeKey(m *message.Message) pb.Entity {
 	return pb.Entity_ENTITY_UNSPECIFIED
 }
 
+// HandleGitHubAppWebhook handles incoming GitHub App webhooks
+func (s *Server) HandleGitHubAppWebhook() http.HandlerFunc {
```

Comment:
There is a fair bit of code duplication between this webhook and the repo webhook. I can either:
1) do a quick refactor now to split the common code into functions
2) have the providers register a handler with the server

Ideally I'd like all github specific code out of controlplane, but maybe that's work for another day?

Reply:
Work for another day, but we should target moving it at some point